### PR TITLE
Rename `parent` to `network` for OVN

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

**Review by tomponline — APPROVED** [review]
LGTM

File: doc/networks.md [hunk 1]
```diff
@@ -267,16 +267,16 @@ Install the OVN tools and configure the OVN integration bridge on the local node
 
 ```
 apt install ovn-host ovn-central
-ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=n.n.n.n \ # The IP of your LXD host on the LAN
+ovs-vsctl set open\_vswitch . \
+  external\_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
```
Comment (tomponline) [inline doc/networks.md:271]: @stgraber these are in a code block and dont need escaping.

Comment (tomponline) [reply doc/networks.md:271]: They render `\_` which is incorrect.

Comment (stgraber) [reply doc/networks.md:271]: Ah, so it's vim getting confused I guess

Comment (stgraber) [reply doc/networks.md]: Dropped that commit

**Pull request merged**